src/components/TreeInfoComponent/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Dispatch, SetStateAction } from "react";
+import { Tree } from "utils/types";
+import styles from "./styles.module.scss";
+interface Props {
+    tree: Tree | undefined;
+    displayingTree: boolean;
+    setDisplayingTree: Dispatch<SetStateAction<boolean>>;
+}
+const TreeInfoComponent: React.FC<Props> = ({tree, displayingTree, setDisplayingTree}) => {
+    console.log(tree);
+    const date = tree && new Date(tree.datePlanted as Date).toLocaleString("en-US", {
+        timeZone: "UTC",
+        weekday: "long",
+        year: "numeric",
+        month: "long",
+        day: "numeric",
+    });
+    return (
+        <div className={`${styles.treeInfo} ${displayingTree ? styles.open : styles.closed}`}>
+            <button className={styles.closeButton} onClick={() => setDisplayingTree(false)}>Close Window</button>
+            <img src={tree && tree.image && tree.image.url} alt={tree.species} className={styles.treeImage} />
```

Comment:
"/defaultTreeImage.png" is the path of the default image that we have been using on the `TreeTable` for trees that don't have an image, probably want to incorporate it here? Also, the images are a bit stretched.
![image](https://user-images.githubusercontent.com/71574118/160886963-898e7ddb-e639-440f-b228-19698f83c329.png)